src/pytorch_lightning/_graveyard/core.py

```diff
@@ -13,25 +13,47 @@
 # limitations under the License.
 from typing import Any
 
-from pytorch_lightning import LightningDataModule
+from pytorch_lightning import LightningDataModule, LightningModule
 
 
 def _on_save_checkpoint(_: LightningDataModule, __: Any) -> None:
     # TODO: Remove in v2.0.0
     raise NotImplementedError(
-        "`LightningDataModule.on_save_checkpoint` was deprecated in v1.6 and removed in v1.8."
+        "`LightningDataModule.on_save_checkpoint` was deprecated in v1.6 and is no longer supported in v1.8."
         " Use `state_dict` instead."
     )
 
 
 def _on_load_checkpoint(_: LightningDataModule, __: Any) -> None:
     # TODO: Remove in v2.0.0
     raise NotImplementedError(
-        "`LightningDataModule.on_load_checkpoint` was deprecated in v1.6 and removed in v1.8."
+        "`LightningDataModule.on_load_checkpoint` was deprecated in v1.6 and is no longer supported in v1.8."
```

Comment:
as of 1.8?

Reply:
Sorry, where do you want to put "as of"?

Reply:
we can also do `1.8+`